package.json

```diff
@@ -5,10 +5,13 @@
   "main": "server.js",
   "type": "module",
   "scripts": {
-    "lint":"eslint --fix \"**/**.js\"",
-    "start":"npm run lint && node server.js",
+    "lint": "eslint --fix \"**/**.js\"",
```

Comment:
Plese fix setup of eslint on project.

steps:

1. change your .eslintrc.json to https://epa.ms/nodejs19-hw2-ex1
2. change parser in .eslintrc.json to @babel/eslint-parser
3. change plugins in .eslintrc.json to
```
 "plugins": [
        "@babel"
    ]
```
4.  change max-len in .eslintrc.json to 
```
        "max-len": [
            2,
            200,
            4,
            {
                "ignoreUrls": true,
                "ignoreComments": false,
                "ignorePattern": "^\\s*(const|let|var)\\s+\\w+\\s+=\\s+\\/.*\\/(|i|g|m|ig|im|gm|igm);?$"
            }
        ],
```
5. add .babelrc file to root of your project 
```
{
  "presets": [
    ["@babel/preset-env",
      {
        "targets": {
          "node": "current"
        }
      }
    ]
  ]
}
```
6. run `npm uninstall babel-register eslint-config-google`
7. run `npm i --save-dev @babel/eslint-parser @babel/eslint-plugin @babel/node`
8. run `npm run lint`
9. fix all issues that described by 8 step